secure_message/resources/threads.py

```diff
@@ -20,14 +22,73 @@ def get(thread_id):
         logger.info("Getting messages from thread", thread_id=thread_id, user_uuid=g.user.user_uuid)
 
         conversation = Retriever().retrieve_thread(thread_id, g.user)
+        conversation_metadata = Retriever.retrieve_conversation_metadata(thread_id)
```

Comment:
Does this return a 404 if the conversation metadata doesn't exist? (it won't exist for all current conversations until they're closed)

Reply:
Now it does yea